lib/util/determine-files.js

```diff
@@ -0,0 +1,81 @@
+let path = require("path");
+let fs = require("fs");
+
+/* Returns a function that determines files to process
+ *
+ * This is useful for pipelines that process a directory of files.
```

Comment:
This doesn't actually mean anything here, does it? As in, we might as well omit this line?